### ISSUE TITLE
`Gcd` no longer returns standard associates?

### ISSUE THREAD

**Issue by ThomasBreuer** [gap-system/gap#3212]
The following happens in GAP 4.10.0 as well as in the current master branch,
with or without packages loaded.

```
gap> x:= Indeterminate( Rationals, "x" );;
gap> g:= Gcd( x-2, x^2-2*x );
-1/2*x+1
gap> StandardAssociate( g );
x-2
```

Up to GAP 4.9, the value returned by `Gcd` was the standard associate.
Thanks to Meinolf Geck for this observation.

I think this changed behavior is a bug, the documentation of `Gcd` says:

> Gcd  returns  the standard associate (see StandardAssociate (56.5-5)) of the
> greatest common divisors.

**Comment by fingolfin** [comment]
This regression was introduced in commit fb0cdaf59f532c8739a1c729d3c1cac1f2228c9c by @hulpke

**Comment by alex-konovalov** [comment]
The fix has been backported to stable-4.10 in commit 9a6034d79f17a529f99cb9379922d98c63d11d9d
